src/components/Description/Description.vue

```diff
@@ -35,21 +35,27 @@
 			@keydown.esc="handleCancelEditing" />
 		<template v-if="!loading">
 			<template v-if="editing">
-				<button class="nc-button nc-button__main description__action"
+				<Button type="tertiary"
+					class="description__action"
 					:aria-label="t('spreed', 'Cancel editing description')"
 					@click="handleCancelEditing">
-					<Close decorative
-						title=""
-						:size="16" />
-				</button>
-				<button class="nc-button nc-button__main primary description__action"
+					<template #icon>
+						<Close decorative
+							title=""
+							:size="16" />
```

Comment:
Also the size of the icons should always be 20, could you please change that?

Reply:
Ok